pkgs/development/interpreters/ngn-k/default.nix

```diff
@@ -31,21 +31,25 @@ useStdenv.mkDerivation {
     # make self-reference for LICENSE
     substituteAllInPlace repl.k
 
+    patchShebangs a19/a.sh a20/a.sh a21/a.sh dy/a.sh e/a.sh
+
     # don't use hardcoded /bin/sh
     for f in repl.k m.c;do
       substituteInPlace "$f" --replace "/bin/sh" "${runtimeShell}"
     done
   '';
 
   makeFlags = [ "-e" ];
-  buildFlags = [ "k" ];
+  buildFlags = [ "k" "libk.so" ];
   checkTarget = "t";
   inherit doCheck;
 
   installPhase = ''
     runHook preInstall
     install -Dm755 k "$out/bin/k"
     install -Dm755 repl.k "$out/bin/k-repl"
+    install -Dm755 libk.so "$out/lib/libk.so"
+    install -Dm644 k.h "$out/include/k.h"
```

Comment:
In that case this would go to `dev`, probably.

Reply:
How do I move it to `dev`?

Reply:
This patch should do the trick:

```diff
diff --git a/pkgs/development/interpreters/ngn-k/default.nix b/pkgs/development/interpreters/ngn-k/default.nix
index 19d50f41370..f4d78a10c5f 100644
--- a/pkgs/development/interpreters/ngn-k/default.nix
+++ b/pkgs/development/interpreters/ngn-k/default.nix
@@ -44,12 +44,14 @@ useStdenv.mkDerivation {
   checkTarget = "t";
   inherit doCheck;
 
+  outputs = [ "out" "dev" "lib" ];
+
   installPhase = ''
     runHook preInstall
     install -Dm755 k "$out/bin/k"
     install -Dm755 repl.k "$out/bin/k-repl"
-    install -Dm755 libk.so "$out/lib/libk.so"
-    install -Dm644 k.h "$out/include/k.h"
+    install -Dm755 libk.so "$lib/lib/libk.so"
+    install -Dm644 k.h "$dev/include/k.h"
     install -Dm644 LICENSE -t "$out/share/ngn-k"
     substituteInPlace "$out/bin/k-repl" --replace "#!k" "#!$out/bin/k"
     runHook postInstall
```

Reply:
Done.